src/app/trade/page.tsx

```diff
@@ -46,11 +53,28 @@ export default function Trade() {
     const intervalId = setInterval(() => {
       dispatch(fetchBalances());
       dispatch(fetchAccountHistory());
+      // console.log("5 seconds");
     }, 5000); // Dispatch every 5000 milliseconds (5 second)
 
     return () => clearInterval(intervalId); // Cleanup interval on component unmount
   }, [dispatch]);
 
+  useEffect(() => {
```

Comment:
Can you add comments to both useEffect's, for example:

// Update orders of selected pair every 5 seconds
// Update orders of all pairs every 2 mins (if selected)